kibana-reports/public/components/context_menu/context_menu.js

```diff
@@ -93,7 +94,7 @@ const generateInContextReport = (
     time_to: timeRanges.time_to.valueOf(),
     report_definition: {
       report_params: {
-        report_name: $('span.euiBreadcrumb').prop('title'),
+        report_name: $('span.euiBreadcrumb').prop('title') + '_' + uuidv4(),
```

Comment:
Do we have snapshot testing for in-context menu UI? Or is there a way to add?

Reply:
We can add snapshot testing- but this line `97` highlighted will not be tested even if we add in-context UI snapshots